documentation/proposals/Proposal - Multi-Backend Input.md

```diff
@@ -834,6 +831,103 @@ public interface IJoystickInputHandler : IInputHandler
 
 `HandleHatMove` must be called when any value of `JoystickState.Hats` changes.
 
+# Input Maps
+
+## Presentation
+Input maps are designed to be easyer to the user to handle actions with more than one input.
+At first view it can looks useless and a bad practice, but this is complemtelly useful for cross-plataform programs, being games an example.
+
+This is an pseudocode example of what the Input Map system solves:
```

Comment:
Problems are best described by words. You haven't stated anywhere in this proposal what this actually achieves and why this is a valuable proposal for the working group to consider.

Reply:
No because it isn't clear what problem you're solving. e.g. what is an input action and why is it useful to hide the actual callbacks behind one?

Reply:
Oh, sure sure! I really have not seen that i should follow that way when i wrote but now i can see it, thx